about/design-goals.md

```diff
@@ -0,0 +1,52 @@
+# CloudEvents Specification Design Goals
+
+CloudEvents are typically used in a distributed system to allow for services to
+be loosely coupled during development, then configured with tight-coupling for
+specific applications.
+
+The goal of the CloudEvents specification is to define interoperability of event
+systems that allow cloud providers, third-party tools and services to provide:
+- new event sources
+- new event consumers
+- observability, such as monitoring, debugging and visualization,
+- developer tools and libraries
+
+## Key properties of the system:
+- Source & action may be developed and deployed independently
+- Trigger configuration allows for actions to be invoked based on a condition
+(e.g. for a specific event type generated by a source, optionally filtered by
+specific attribute values, such as a Function which is invoked when an image is
+uploaded to specific S3 or Google Cloud Storage “bucket” path)
+- Events are uniquely identified to facilitate developer velocity and tooling:
+  - a unique idenfier allows for idempotency for reliable implementation
+  - structured event definitions allow the consumer to rely on a specific format
+- Common metadata attributes allow for observability via tools whithout
+knowledge of the specific implementation of an event system and also can provide
+value without detailed handling of the specifics of the unique event data
+- CloudEvents may be transported from source to action with different protocols.
+
+In order to achieve the above goals and desired system properties, the
+Serverless WG must agree on a common vocabulary with precise definitions
+describing the key elements of the system including:
+- [Source](../spec.md#source)
+- Action <use definition from starter doc>
+- Trigger: the condition that causes an event to be transmitted from source to
+action
+- Rule: the association of trigger + action
+
+In order to achieve the interoperability goals, the Serverless WG must describe:
```

Comment:
I would merge this paragraph with the previous one and just have one list of "goals" rather than split it into two. I would also avoid defining the terms here since we're going to do that later on (if we merge this into the spec). I would just state the goal itself - meaning, something like: Define the common vocabulary of an eventing system